### PR TITLE
Edited tooltip text for accuracy and grammar.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/12893.

### CODE REVIEW

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -198,9 +198,9 @@ If you're feeling frisky, click yourself in help intent to pull the object out."
 /obj/screen/alert/weightless
 	name = "Weightless"
 	desc = "Gravity has ceased affecting you, and you're floating around aimlessly. You'll need something large and heavy, like a \
-wall or lattice structure, to push yourself off of if you want to move. A jetpack would enable free range of motion. A pair of \
```
Comment (ChangelingRain) [inline code/_onclick/hud/alert.dm]: This is correct, though 'lattice structure' is redundant.

File: code/_onclick/hud/alert.dm [hunk 2]
```diff
@@ -238,12 +238,12 @@ or shoot a gun to move around via Newton's 3rd Law of motion."
 /obj/screen/alert/emptycell
 	name = "Out of Power"
 	desc = "Unit's power cell has no charge remaining. No modules available until power cell is recharged. \
-Recharging stations are available in robotics, the dormitory's bathrooms. and the AI satellite."
+Recharging stations are available in robotics, the dormitory bathrooms. and the AI satellite."
```
Comment (ChangelingRain) [inline code/_onclick/hud/alert.dm]: Comma, not period.

File: code/_onclick/hud/alert.dm [hunk 3]
```diff
@@ -238,12 +238,12 @@ or shoot a gun to move around via Newton's 3rd Law of motion."
...
 	icon_state = "emptycell"
 
 /obj/screen/alert/lowcell
 	name = "Low Charge"
-	desc = "Unit's power cell is running low. Recharging stations are available in robotics, the dormitory's bathrooms. and the AI satellite."
+	desc = "Unit's power cell is running low. Recharging stations are available in robotics, the dormitory bathrooms. and the AI satellite."
```
Comment (ChangelingRain) [inline code/_onclick/hud/alert.dm]: See above.

File: code/_onclick/hud/alert.dm [hunk 4]
```diff
@@ -140,7 +140,7 @@ The box in your backpack has an oxygen tank and gas mask in it."
 
 /obj/screen/alert/starving
 	name = "Starving"
-	desc = "Some food would be to kill for right about now. The hunger pains make moving around a chore."
+	desc = "You're severely undernourished. The hunger pains make moving around a chore."
```
Comment (PKPenguin321) [inline code/_onclick/hud/alert.dm]: >undernourished

malnourished

File: code/_onclick/hud/alert.dm (hunk 2 above)
Comment (bgobandit) [inline code/_onclick/hud/alert.dm]: fuck I missed that one I fail at editing

Comment (neersighted) [inline code/_onclick/hud/alert.dm]: Unit power cell.

**Comment by optimumtact** [comment]
I kinda miss the russianisms

File: code/_onclick/hud/alert.dm [hunk 5]
```diff
@@ -310,7 +310,7 @@ so as to remain in compliance with the most up-to-date laws."
 
 /obj/screen/alert/restrained/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click the alert to unbuckle unless you're handcuffed."
+	desc = "You've been buckled to something. Click the alert to unbuckle unless you're handcuffed."
```
Comment (MrPerson) [inline code/_onclick/hud/alert.dm]: Part of the goal of the alert was to explain why you're unable to move. Removing that seems counterproductive.

Comment (bgobandit) [inline code/_onclick/hud/alert.dm]: The problem is you aren't always unable to move, see https://github.com/tgstation/-tg-station/issues/12893

Comment (MrPerson) [inline code/_onclick/hud/alert.dm]: Ideally those cases just wouldn't throw the normal buckled alert. Of course I understand that the underlying code is all so mixed together that having such a special case is an unreasonable request right now.

Maybe make the text just say "Can't walk normally."? That should cover all cases fairly well. That might still imply "can't move" too much so just do whatever.

**Pull request merged**